api/src/lib/sentry/index.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from 'path';
+import Raven from 'raven';
+import { errResponse } from '../utils/response';
+
+const parseUser = (event: AWSLambda.APIGatewayProxyEvent): SentryUserInfo => {
+  if (!event.requestContext.authorizer) { return; }
+
+  const authorizerUser = JSON.parse(event.requestContext.authorizer.user);
+  return {
+    email: authorizerUser.email,
+    username: authorizerUser.username
+  };
+};
+
+const getSourceMaps: Raven.DataCallback = (data: {[key: string]: any}): {[key: string]: any} => {
+  const stacktrace: any = data.exception && data.exception[0].stacktrace;
+  if (stacktrace && stacktrace.frames) {
+    stacktrace.frames.forEach((frame: any) => {
+      const p = path.relative('/var/task/', frame.filename);
+      frame.filename = 'app:///' + p.split('/')[0] + '/' + p;
+    });
```

Comment:
I took this from the [Sentry docs](https://docs.sentry.io/clients/node/typescript/) (the example is at the very end) and adapted it a little. Since we're building the sources into `hello/hello/hello.js`, Sentry expects the source map name `~/hello/hello/hello.map.js`. The lambda handler is `hello/hello.js`, so the path of the frame would be `/var/task/hello/hello.js`; we're stripping away the `/var/task` part and adding an additional `hello` to match the pattern Sentry expects.